mungegithub/mungers/close-stale.go

```diff
@@ -109,14 +98,21 @@ func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool)
 	return lastHuman, true
 }
 
-func findLastHumanIssueUpdate(obj *github.MungeObject) (*time.Time, bool) {
-	lastHuman := obj.Issue.CreatedAt
+func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	pr, ok := obj.GetPR()
```

Comment:
I realize that you're just moving code here, but you don't need to get the PR to get `CreatedAt`